pkgvars

```diff
@@ -2,3 +2,4 @@ pkgname=citus
 pkgdesc='Citus (Open-Source)'
 pkglatest=9.5.10.citus-1
 versioning=fancy
+nightlyref=notice_test
```

Comment:
Do we plan to keep this line here? Is it for testing purposes?

BTW, missing EOL character at the end.

Reply:
It is for testing purpose since the master branch does not include NOTICE file. I'm opening a PR to add Notice file into master branch of citus

Reply:
We have the file on master. See https://github.com/citusdata/citus/blob/master/NOTICE

Reply:
Could we add 9.x release branches? I got error because there is not a NOTICE file for release branches?

Reply:
Branch PR's
https://github.com/citusdata/citus/pull/5479
https://github.com/citusdata/citus/pull/5478
https://github.com/citusdata/citus/pull/5477
https://github.com/citusdata/citus/pull/5476